lib/diaspora_federation.rb

```diff
@@ -22,10 +22,11 @@ module DiasporaFederation
     fetch_person_for_hcard
     save_person_after_webfinger
     fetch_private_key_by_id
-    fetch_private_key_by_post_guid
+    fetch_private_key_by_entity_guid
     fetch_public_key_by_id
-    fetch_public_key_by_post_guid
-    post_author_is_local?
+    fetch_public_key_by_entity_guid
```

Comment:
I also thought about the README.md, but I like the `DiasporaFederation` module more. I don't want the readme too big, because it should tell some basic info about the gem/project and the technical documentation is in the [rubydoc](http://www.rubydoc.info/gems/diaspora_federation/) then.